### DISCUSSION TITLE
Disconnection issue

### DISCUSSION THREAD

**Discussion by fabio1605** [rsjaffe/MIDI2LR#806] [Problems]
Hello - when i am using the plugin it works great - but if i put my pc to standby for example when ressuming standby the midi 2lr stops working.

I need to disconnect/reconnnect the usb a few times - stop and start the software a few times before it comes back to life.

Any ideas

**Comment by antzpantz** [comment]
I'm sure that's more a USB/controller issue than a MIDI2LR issue. Just
close MIDI2LR/Lightroom before putting your PC on standby.

**Comment by fabio1605** [comment]
I don't think it's a usb controller issue. The midi device is still detected it just midi2lr doesn't seem to pick it up. I have to close and start the software a few times. 
-------- Original message --------From: antzpantz ***@***.***> Date: 12/12/2024  19:21  (GMT+00:00) To: rsjaffe/MIDI2LR ***@***.***> Cc: fabio1605 ***@***.***>, Author ***@***.***> Subject: Re: [rsjaffe/MIDI2LR] Disconnection issue (Discussion #806) 
I'm sure that's more a USB/controller issue than a MIDI2LR issue. Just

close MIDI2LR/Lightroom before putting your PC on standby.
